heft-plugins/heft-lint-plugin/src/Eslint.ts

```diff
@@ -227,6 +229,21 @@ export class Eslint extends LinterBase<TEslint.ESLint.LintResult> {
         }
       }
     }
+
+    const sarifLogPath: string | undefined = this._sarifLogPath;
+    if (sarifLogPath) {
+      const { formatAsSARIF } = await import('./SarifFormatter');
+      const sarifString: string = JSON.stringify(
+        formatAsSARIF(lintResults, {
+          ignoreSuppressed: false,
```

Comment:
Is this something that should be configurable?

Reply:
For the applications we want this for, reporting suppressed rules is desired, so we could probably just hardcode this to `false` all the way through, but for now we're leaving the option open.